packages/helper-grammar-regex-collection/test.js

```diff
@@ -120,7 +123,8 @@ const fixtures = {
       ['var _ = import("foo")', ['foo']],
       ['var foo = import("foo")var bar = import("bar")', ['foo', 'bar']],
       ['import("foo")import("bar");', ['foo', 'bar']],
-      ['import("foo")import("bar");', ['foo', 'bar']],
+      ['import("foo")import("bar")', ['foo', 'bar']],
```

Comment:
Can you change this line back to how it was before? I'm assuming it was an accidental change.

Reply:
Sorry, I should have made a comment on this line.

I saw that this line was identical to the line above it, and guessed that it was meant to be like lines 128 and 129, where the first line had a semicolon and the second did not.

My bad for just changing it in this pull request.  Would you like the change rolled back, line 126 removed entirely, or the change left to avoid having two identical assertions?

Reply:
Oh, I hadn't even noticed that, thanks for explaining! I think it's fine to leave it as it currently.